tools/repl/src/repl.rs

```diff
@@ -162,18 +162,14 @@ impl REPL {
         let service_id = uuid::Uuid::new_v4().to_string();
 
         let start = Instant::now();
-        let app_service = match config_file_path {
-            Some(config_file_path) => {
-                let config_file_path = config_file_path.into();
-                AppService::with_raw_config(config_file_path, &service_id, Some(&tmp_path))
-            }
-            None => {
-                let mut config: fluence_app_service::RawModulesConfig = <_>::default();
-                config.service_base_dir = Some(tmp_path);
 
-                AppService::new(std::iter::empty(), config, &service_id)
-            }
-        }?;
+        let mut config = config_file_path
+            .map(|p| RawModulesConfig::load(p.into()))
+            .unwrap_or_default();
```

Comment:
Otherwise it woun't compiled.

```suggestion
        let mut config = config_file_path
            .map(|p| RawModulesConfig::load(p.into()))
            .transpose()?
            .unwrap_or_default();
```